src/index.ts

```diff
@@ -130,6 +131,7 @@ class ChildAPI {
    * @prop {promise} handshake	- Promise that is waiting for connection confirmation
    * @prop {string} class_name	- iFrame's unique class name
    * @prop {boolean} loaded		- Indicates if iFrame successfully loaded
+   * @prop {any} signalCb - A callback that's run when we receive a signal
```

Comment:
The whitespace is off here.

Reply:
Yes, this might be the right moment to get formatting done right @robbiecarlton